functions/src/userManagement.ts

```diff
@@ -63,6 +64,8 @@ export default class UserManagement {
       return;
     }
 
+    accountData.uid = uid
```

Comment:
as mentioned, I think we should just not try to duplicate uid (remove it from `keysToDuplicate` instead

Reply:
I agree. I just don't want to break anything else, so I'll leave as is for now. I verified that it does work now